### ISSUE TITLE
[Rule change] `prefer-extracting-callbacks` allow short callbacks to exist

### ISSUE THREAD

**Issue by fzyzcjy** [dart-code-checker/dart-code-metrics#873]
That should be a configurable line count. For example, when max-line-count is configured to 5, a callback with <5 lines of code will not be linted. This ensures that very short callback code can exist there, which can make code quite clean.

**What rule do you want to change?**

**How will the change be implemented? (New option, new default behavior, etc.)?**

**Please provide some example code that this change will affect:**

```dart
```

**What does the rule currently do for this code?**

**What will the rule do after it's changed?**

**Are you willing to submit a pull request to implement this change?**

**Comment by incendial** [comment]
@fzyzcjy sounds great, actually, thanks for the suggestion!

**Comment by fzyzcjy** [comment]
implemented.
